packages/fxa-event-broker/lib/notificationProcessor.ts

```diff
@@ -130,85 +87,93 @@ class ServiceNotificationProcessor {
     this.webhookService.stop();
   }
 
+  private async handleDeleteEvent(message: deleteSchema) {
+    const clientIds = await this.db.fetchClientIds(message.uid);
+    clientIds.forEach(async clientId => {
```

Comment:
Looking again at this function, the async/foreach thing pops up here too. I think a simple for loop would allow the async calls to be managed correctly with minimal rewriting.

Reply:
Ah yes, fixed that to use a for..of loop.